### PR TITLE
server using flask framework

### PR DESCRIPTION
**Pull request by abhishek1995s**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,25 +0,0 @@
-FROM node:boron
```
Comment (niranjan94) [inline Dockerfile]: We need docker deployment too.

File: Procfile [hunk 2]
```diff
@@ -1,2 +0,0 @@
-pipinstall: pip install -r requirements.txt
```
Comment (niranjan94) [inline Procfile]: We need heroic deployment too

File: app.json [hunk 3]
```diff
@@ -1,24 +0,0 @@
-{
-  "name": "query-server",
```
Comment (niranjan94) [inline app.json]: Required for heroic

File: package.json [hunk 4]
```diff
@@ -1,23 +0,0 @@
-{
-  "name": "query-1",
-  "version": "0.0.0",
-  "scripts": {
-    "start": "node src/app.js"
-  },
```
Comment (niranjan94) [inline package.json]: Required for bootstrap and prisms

File: server.py [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+app=Flask(__name__)
+@app.route('/')
+def initial():
+    
+        return render_template('index.html',result=' ')
+    #else:
+        #return render_template('newmenu.html',restaurant_id=restaurant_id)  
+@app.route('/query', methods=['GET', 'POST'])
+def query_processor():
+        query=request.form['query']
```
Comment (enigmaeth) [inline server.py]: I see there is no connection made to the mongo database. Please implement that along with the changed schema. 
Where are you storing the query string?

**Comment by abhishek1995s** [comment]
@niranjan94 api is  ready for both xml and json. flask has some issue in syntax highlighting of xml feed. I will fix it soon. At current point it is easy to show json format(issue #46 )

**Comment by abhishek1995s** [comment]
will fix the syntax highlighting as soon as i can

**Comment by niranjan94** [comment]
Closing in favor of #48

**Pull request closed without merging**